src/Ozone_Monitor/image_gen/ush/plot_horiz.sh

```diff
@@ -41,61 +40,72 @@ while [[ $ctr -le 3 ]]; do
    c_pdy=`echo $cdate|cut -c1-8`
    c_cyc=`echo $cdate|cut -c9-10`
 
-   tankdir_cdate=${TANKDIR}/${RUN}.${c_pdy}/${c_cyc}/atmos/oznmon/horiz
+   tankdir_cdate=${OZN_TANKDIR_STATS}/${RUN}.${c_pdy}/${c_cyc}/atmos/oznmon/horiz
    if [[ ! -d ${tankdir_cdate} ]]; then
-      tankdir_cdate=${TANKDIR}/${RUN}.${c_pdy}/${c_cyc}/oznmon/horiz
+      tankdir_cdate=${OZN_TANKDIR_STATS}/${RUN}.${c_pdy}/${c_cyc}/oznmon/horiz
       if [[ ! -d ${tankdir_cdate} ]]; then
-         tankdir_cdate=${TANKDIR}/${RUN}.${c_pdy}/horiz
+         tankdir_cdate=${OZN_TANKDIR_STATS}/${RUN}.${c_pdy}/horiz
       fi
    fi
 
-   $NCP ${tankdir_cdate}/${SATYPE}.${dsrc}.ctl ./
-   $NCP ${tankdir_cdate}/${SATYPE}.${dsrc}.${c_pdy}* ./
+   if [[ -e ${tankdir_cdate}/${SATYPE}.${dsrc}.ctl ]]; then 
+      $NCP ${tankdir_cdate}/${SATYPE}.${dsrc}.ctl ./
+   fi
+
+   if compgen -G "${tankdir_cdate}/${SATYPE}.${dsrc}.${c_pdy}*ieee_d*" > /dev/null; then
+      $NCP ${tankdir_cdate}/${SATYPE}.${dsrc}.${c_pdy}*ieee_d* ./
```

Comment:
I am sometimes getting errors here that a *gz file do not exist when attempting to copy.  I am not sure, but it seems that these files are being compressed/decompressed somewhere else in the script(s), so between the `compgen` command and the `$NCP` command, the filename changes.

Reply:
Hmmm.  That's interesting.  I'll reexamine my logs and see if I have similar errors reported.

Reply:
Oh, I think I see the problem.  Outside of the while loop where that error is reported, there is this

```
if compgen -G "*.gz" > /dev/null; then
   $UNCOMPRESS *.gz
fi
```
So if one of the treads is ahead of others we can hit that problem.

Reply:
If I make the file selection specific to $SATYPE that should fix it.

Reply:
Nope, that's not it and I'm being dim -- that couldn't be the problem anyway because the uncompress is being done on the working directory not the data source directory.